secant/Screens/Onboarding/OnboardingFooterView.swift

```diff
@@ -13,61 +13,85 @@ struct OnboardingFooterView: View {
     let animationDuration: CGFloat = 0.8
 
     var body: some View {
-        GeometryReader { proxy in
-            WithViewStore(self.store) { viewStore in
-                VStack(spacing: 5) {
-                    Spacer()
+        WithViewStore(self.store) { viewStore in
+            VStack(spacing: 5) {
+                Spacer()
+                
+                if viewStore.isFinalStep {
+                    Button("Create New Wallet") {
+                        withAnimation(.easeInOut(duration: animationDuration)) {
+                            viewStore.send(.createNewWallet)
+                        }
+                    }
+                    .createButtonStyle
+                    .buttonLayout()
                     
-                    if viewStore.isFinalStep {
-                        Button("Create New Wallet") {
-                            withAnimation(.easeInOut(duration: animationDuration)) {
-                                viewStore.send(.createNewWallet)
-                            }
+                    Button("Import an Existing Wallet") {
+                        withAnimation(.easeInOut(duration: animationDuration)) {
+                            viewStore.send(.createNewWallet)
                         }
-                        .primaryButtonStyle
-                        .frame(height: proxy.size.height / 12)
-                        .padding(.horizontal, 15)
-                        .transition(.opacity)
-                    } else {
-                        Button("Next") {
-                            withAnimation(.easeInOut(duration: animationDuration)) {
-                                viewStore.send(.next)
-                            }
+                    }
+                    .secondaryButtonStyle
+                    .buttonLayout()
+                } else {
+                    Button("Next") {
+                        withAnimation(.easeInOut(duration: animationDuration)) {
+                            viewStore.send(.next)
                         }
-                        .primaryButtonStyle
-                        .frame(height: proxy.size.height / 12)
-                        .padding(.horizontal, 15)
-                        .transition(.opacity)
                     }
+                    .primaryButtonStyle
+                    .buttonLayout()
                     
                     ProgressView(
-                        "\(viewStore.index + 1)/\(viewStore.steps.count)",
+                        "0\(viewStore.index + 1)",
                         value: Double(viewStore.index + 1),
                         total: Double(viewStore.steps.count)
                     )
-                    .onboardingProgressStyle
```

Comment:
If the view in question has a trailing paren at the same indentation level as the view declaration, then we don't indent the view modifiers.  This is sometimes called `dangling modifiers` and we try to avoid them when possible.

Reply:
fixed